server/testdb/transactions_test.go

```diff
@@ -233,82 +233,3 @@ func TestGetTransactionsWithMultipleVersions(t *testing.T) {
 		assert.Equal(t, 2, test.Version)
 	}
 }
-
-// func TestTransactionSummary(t *testing.T) {
```

Comment:
can all this commented code be deleted?

Reply:
I forgot to remove it in my old PR. We don't have transaction summaries yet, so I decided to remove it and reintroduce it later.